packages/admin-panel/src/pages/resources/ResourcePage.jsx

```diff
@@ -128,13 +171,18 @@ ResourcePage.propTypes = {
   defaultSorting: PropTypes.array,
   defaultFilters: PropTypes.array,
   editorConfig: PropTypes.object,
+  detailsView: PropTypes.object,
+  parent: PropTypes.object,
+  displayProperty: PropTypes.string,
+  getIsLink: PropTypes.func,
```

Comment:
```suggestion
  getHasDetailsView: PropTypes.func,
```
Links are always tied to a `detailsView` yeah? Might be nice to link these two concepts with a single term so when we're refactoring it's easy to know they're part of the same logic flow.